### ISSUE TITLE
Clarify MASA response format requirements

### ISSUE THREAD

**Issue by EskoDijk** [anima-wg/anima-brski-prm#48]
> The expected voucher-response format is indicated by the Accept
>    header field of the RVR or MASA SHOULD respond with the same format
>    as the PVR was (default "JSON-in-JWS")

It provides 2 options "X or Y". Unclear what advice to follow. Maybe needs a priority like:
1) MASA SHOULD respond with the 'Accept' indicated format, if it supports it. ( Regardless of whether the Pledge would support that format ?)
2) If no 'Accept' header included, then respond with format of PVR.

**Comment by stfries** [comment]
As the pledge determines the final format Option 2 is actually the way to go. Will reflect this in the text. 
Also, the Registrar should signal in the Accept header the format the pledge understands. If there is a deviation between format in accept header and the format of the request object, the request object wins and a warning may be issued.

**Comment by EskoDijk** [comment]
Text doesn't read correctly, I think.   And if Registrar should not include any Accept header, the text should clarify this. If the Registrar should include an Accept header, text should also say this I think.

If Accept header is allowed to be sent by Registrar: then if the MASA knows the the requested format by Registrar (based on Accept header) will not be parseable by the Pledge because it's in the wrong format, it SHOULD send a 406 Not Acceptable.

**Comment by stfries** [comment]
Agree. I enhanced the proposed text to 

The expected voucher-response format for the pledge-responder-mode the `application/voucher-jws+json` as defined in {{I-D.ietf-anima-jws-voucher}} is applied.
If the MASA detects that the Accept header of the PVR does not match the `application/voucher-jws+json` it SHOULD respond with the HTTP error code 406 as the pledge will not be able to parse the response.

This should address the comment. Closing?
